back/engines/commercial/user_custom_fields/spec/policies/user_custom_fields/representativeness/ref_distribution_policy_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserCustomFields::Representativeness::RefDistributionPolicy, type: :policy do
+  subject(:policy) { described_class.new(user, ref_distribution) }
+
+  let_it_be(:ref_distribution) { create(:ref_distribution) }
```

Comment:
Why don't you use `let` instead of `let_it_be" in this file?

Reply:
It makes the tests faster.

Reply:
How much faster? Why don't we use it in all specs if it makes the tests faster?

Reply:
In this case, I think the gain is around 20%. But it greatly varies depending on the test file. It's a pretty recent addition to the code base (around September 2021, I'd say) and I don't think everyone knows about it or feels comfortable enough using it (we cannot always use this trick).

Reply:
Seems to be a topic for the CDM :-)

Reply:
https://docs.google.com/document/d/1zcqyfubUoWikpzf1PzLcTEFI9MbAhZ9S5FbMqkkcQMc/edit?skip_itp2_check=true#heading=h.cc5vwaqbjg57 :)